packages/vscode-container-client/src/clients/DockerClientBase/DockerInspectContainerRecord.ts

```diff
@@ -123,16 +123,15 @@ export function normalizeDockerInspectContainerRecord(container: DockerInspectCo
                     source: mount.Source,
                     destination: mount.Destination,
                     readOnly: !mount.RW,
-                }];
+                } satisfies InspectContainersItemBindMount];
             case 'volume':
                 return [...curMounts, {
                     type: 'volume',
-                    name: mount.Name,
-                    source: mount.Source,
+                    source: mount.Name,

```

Comment:
This was a longstanding bug. The description of `source` on `InspectContainersItemVolumeMount` is `The source of the volume mount (volume name)`, we should have been using `mount.Name`, not `mount.Source` (which is an obscure path deep in the Docker data dirs and not human-friendly).

The same bug existed for the Podman client.